packages/typegen/src/index.ts

```diff
@@ -114,7 +114,7 @@ export const generate = async (params: Partial<Options>) => {
       return itemType.match(/^\w+$/) ? `${itemType}[]` : `Array<${itemType}>`
     }
 
-    if (regtype.match(/\(\d+\)/)) {
+    if (regtype.match(/\([\d,]+\)/)) {
```

Comment:
Should probably allow for whitespace too

Reply:
Ok done :)